skxray/roi.py

```diff
@@ -264,6 +264,8 @@ def segmented_rings(edges, segments, center, shape, offset_angle=0):
     """
     agrid = core.angle_grid(center, shape)
 
+    agrid[agrid < 0] = 2*np.pi + agrid[agrid < 0]
```

Comment:
Good catch. This was indeed broken, but I think this solution is correct in every case. I will merge your PR onto this PR, add a test to demonstrate the problem I am seeing, and propose a fix in a new commit.